src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -81,7 +81,9 @@ def __init__(self, config):
 
         # self.LayerNorm is not snake-cased to stick with TensorFlow model variable name and be able to load
         # any TensorFlow checkpoint file
-        self.LayerNorm = nn.LayerNorm(config.hidden_size, eps=config.layer_norm_eps)
+        self.normalize_embeddings = config.normalize_embeddings
```

Comment:
Thanks a lot for the PR @Soonhwan-Kwon ! for `transformers` we have a rather strict rule to not adapt existing modeling files for new modeling checkpoints so in this case here it would be great if you could create a new `modeling_xlm_roberta_xl.py` file

Reply:
Thank you fort the review, and I began to make xlm_roberta_xl as your suggestion.